src/views/Home2/components/CompositeImage.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import styled, { keyframes } from 'styled-components'
+import { Box } from '@pancakeswap/uikit'
+
+const floatingAnim = (x: string, y: string) => keyframes`
+  from {
+    transform: translate(0,  0px);
+  }
+  50% {
+    transform: translate(${x}, ${y});
+  }
+  to {
+    transform: translate(0, 0px);
+  }  
+`
+
+const Wrapper = styled(Box)<{ maxHeight: string }>`
+  position: relative;
+  max-height: ${({ maxHeight }) => maxHeight};
+
+  & :nth-child(2) {
+    animation: ${floatingAnim('3px', '15px')} 3s ease-in-out infinite;
+    animation-delay: 1s;
+  }
+
+  & :nth-child(3) {
+    animation: ${floatingAnim('5px', '10px')} 3s ease-in-out infinite;
+    animation-delay: 0.66s;
+  }
+
+  & :nth-child(4) {
+    animation: ${floatingAnim('6px', '5px')} 3s ease-in-out infinite;
+    animation-delay: 0.33s;
+  }
+
+  & :nth-child(5) {
+    animation: ${floatingAnim('4px', '12px')} 3s ease-in-out infinite;
+    animation-delay: 0s;
+  }
+`
+
+const DummyImg = styled.img<{ maxHeight: string }>`
```

Comment:
what is that ?

Reply:
All of the visible images are absolutely positioned. The dummy img is a tool for having a same-sized image that _is_ part of the document flow (so that other elements' positioning/margins etc are easy to manage). 

The image src & srcset is the same as a visible image so it doesn't result in the image being downloaded twice.